UnitedIncome/components
# [APP-5098] Tile Component

### Description
> Please provide a brief description of your changes below.

Implements the initial base `Tile` component. It supports `children` and a `footerContent` prop.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Take a look at it, it's basically just a box with a border.
* Because the idea is for this component to support any content type, there's no need for it to enforce any sort of styling. You should assign a class to the item before you pass it in to this component. 

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

<img width="1457" alt="Screen Shot 2020-05-04 at 3 21 17 PM" src="https://user-images.githubusercontent.com/10888441/81004887-140ed900-8e1b-11ea-89b7-ab2130737eac.png">
<img width="1458" alt="Screen Shot 2020-05-04 at 3 21 25 PM" src="https://user-images.githubusercontent.com/10888441/81004889-14a76f80-8e1b-11ea-9d1b-ae77ffee1528.png">


### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-5098)
